### PR TITLE
Import C# solutions from https://github.com/kfstorm/LeetCode

### PR DESCRIPTION
**Pull request by kfstorm**
Here is the importing script: https://gist.github.com/kfstorm/0f35d62c68ae4bf9c8272711e8abd99a

### CODE REVIEW

File: solution/0001.Two Sum/Solution.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+public class Solution {
+    public int[] TwoSum(int[] nums, int target) {
+        var dict = new Dictionary<int, int>();
+        for (var i = 0; i < nums.Length; ++i)
+        {
+            int index;
+            if (dict.TryGetValue(target - nums[i], out index))
+            {
+                return new [] { index + 1, i + 1};
```
Comment (yanglbme) [inline solution/0001.Two Sum/Solution.cs]: Expected [0, 1], but got [1, 2]:

image


```suggestion
                return new [] { index, i};
```

Comment (kfstorm) [reply solution/0001.Two Sum/Solution.cs]: Maybe my solution is too old and the test cases have updated. Let me run all my solutions on LeetCode again. Will update the PR later.

Comment (yanglbme) [reply solution/0001.Two Sum/Solution.cs]: Thank you :+1:

Comment (kfstorm) [reply solution/0001.Two Sum/Solution.cs]: Updated.

**Comment by kfstorm** [comment]
All solutions were just tested.

**Comment by yanglbme** [comment]
Merged.

**Pull request merged**